From ac99388ddb8d37f8e3e0172d28827317b34cda24 Mon Sep 17 00:00:00 2001
From: Li Jun <b47624@freescale.com>
Date: Thu, 16 Oct 2014 22:17:57 +0800
Subject: [PATCH 0140/5242] MLK-9618-4 usb: chipidea: otg: show message for
 a_wait_bcon time out

commit  da68fcf3b12badd0b1a6ad9a0c71bda1d55a5a73 from
https://source.codeaurora.org/external/imx/linux-imx.git

When a host start a session and wait b device to connnect within
TA_WAIT_BCON, if failed, should show a message 'Device no response'
according to otg and EH compliance plan version 1.2 section 6.7.22

Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 1762592ead672d27b281df8becf90f8c7d7fe61c)
(cherry picked from commit f1f68bc5f62848382646599c3609ad797bf3b5a5)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/chipidea/otg_fsm.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/chipidea/otg_fsm.c b/drivers/usb/chipidea/otg_fsm.c
index 8180c9c..4107c42 100644
--- a/drivers/usb/chipidea/otg_fsm.c
+++ b/drivers/usb/chipidea/otg_fsm.c
@@ -304,6 +304,7 @@ static int a_wait_vfall_tmout(struct ci_hdrc *ci)
 static int a_wait_bcon_tmout(struct ci_hdrc *ci)
 {
 	ci->fsm.a_wait_bcon_tmout = 1;
+	dev_warn(ci->dev, "Device No Response\n");
 	return 0;
 }
 
-- 
1.7.9.5

